build/util/Dockerfile

```diff
@@ -1,11 +1,11 @@
-# Argo CD v2.2.2
-FROM quay.io/argoproj/argocd@sha256:358c244c96313ca3bf9f588dc870d8123fc22ffa5c231c57da10f77b8d671c66
+# Argo CD v2.3.3
+FROM quay.io/argoproj/argocd@sha256:dd738f234fcdb0aac8631a0fd1aafbbcd86f936480b06e8377b033ef7a764f71
 
 USER root
 
 # Ensure system dependencies are installed
 RUN apt-get update && \
-    apt-get install -y curl && \
+    apt-get install -y curl python3-pip && \
```

Comment:
Please check to see if this is needed.

Reply:
ArgoCD removed python from their image in 2.3, thus we need to add it here.

Reply:
I remember I raised this question with @iam-veeramalla . He was able to build without the python3 dependency. It would be nice to pull in just what we need. If we indeed need it, then obviously we should have it. What is error that you see after you remove it?

Reply:
```/home/argocd/google-cloud-sdk/install.sh
Welcome to the Google Cloud CLI!
WARNING: You appear to be running this script as root. This may cause 
the installation to be inaccessible to users other than the root user.
/home/argocd/google-cloud-sdk/install.sh: 210: python: not found
The command '/bin/sh -c curl -sL https://sdk.cloud.google.com/ > /tmp/install.sh &&     bash /tmp/install.sh --disable-prompts --install-dir=/home/argocd &&     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*' returned a non-zero code: 127
make: *** [util-build] Error 127```